WooCommerce/Classes/Extensions/OrderAttributionInfo+Origin.swift

```diff
@@ -0,0 +1,71 @@
+import Foundation
+import struct Yosemite.OrderAttributionInfo
+
+/// Provides `Origin` value for order attribution
+///
+/// Implementation based on https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Internal/Traits/OrderAttributionMeta.php#L276-L314
+///
+extension OrderAttributionInfo {
+    var origin: String {
+        switch sourceType {
+        case "utm":
+            return String.localizedStringWithFormat(Localization.source, source ?? "")
+        case "organic":
+            return String.localizedStringWithFormat(Localization.organic, source ?? "")
+        case "referral":
+            return String.localizedStringWithFormat(Localization.referral, source ?? "")
```

Comment:
Since `source` here is optional, what about providing a Localized string in case of an empty state (Eg. Unknown itself)?

Reply:
Good suggestion. Done in https://github.com/woocommerce/woocommerce-ios/pull/11963/commits/d9ee223b4ab5a31603eafe00f7cacd8b73fa7773